src/main/java/nextstep/courses/domain/Course.java

```diff
@@ -50,4 +51,13 @@ public String toString() {
                 ", updatedAt=" + updatedAt +
                 '}';
     }
+
+    public Long addSession(Session session) {
+        this.sessionList.addSession(session);
```

Comment:
```suggestion
        this.sessions.add(session);
```

저는 콜렉션은 습관적으로 `-s` 패턴으로 작성하는편이에요.
이유는 자료타입을 이름으로 명시하는게 사용할때 저의 생각을 방해하는 요소가 되기도하고, 
타입이 `map`,`set`으로 변경되면 이름도 변경해줘야 하는 문제가 발생하기 때문이에요.

부르는 입장에서 자료구조는 궁금해하지 않았으면 해요.
왜냐하면 그냥 `session`가 알아서 추가해주길 바랄것 같기 때문이죠.

Reply:
자료구조이름을 제외하고 sessions로 변경하였습니다!